cpp/daal/src/externals/core_threading_win_dll.cpp

```diff
@@ -41,7 +41,11 @@ daal::services::Environment::LibraryThreadingType __daal_serv_get_thr_set();
 
 DAAL_EXPORT HMODULE load_onedal_thread_dll()
 {
-    return DAAL_LOAD_DLL(_DLL_SUFFIX("onedal_thread"));
```

Comment:
why is this no longer working?

Reply:
Idk, but in ICX the concatenate operation ## works pretty strange and MKL function macros have been updated for it.

Reply:
I'd recommend reverting changes to this if possible - it seems like the original should be working (just this deletion and 4 lines of additions)

Reply:
It doesnt work